packages/core/src/spam-protection/spam-protection-action-creator.ts

```diff
@@ -22,14 +22,28 @@ export default class SpamProtectionActionCreator {
     initialize(
         options?: SpamProtectionOptions,
     ): ThunkAction<SpamProtectionAction, InternalCheckoutSelectors> {
-        return (store) =>
-            concat(
+        return (store) => {
+            const state = store.getState();
+            const isRecaptchaResetExperimentEnabled =
+                state.config.getConfig()?.storeConfig.checkoutSettings.features[
+                    'CHECKOUT-8264.recaptcha_error_fix'
+                ] ?? false;
```

Comment:
Perhpas we can build the similar function in sdk and ensure the default value is `true`.
```
export default function isExperimentEnabled(
    checkoutSettings: CheckoutSettings | undefined,
    experimentName: string,
): boolean {
    return Boolean(checkoutSettings?.features[experimentName] ?? true);
}
```

Reply:
SDK is different than JS, as folks don't fork it so we should be ok in that regard.

Reply:
On a second thought, when a forked custom checkout-js uses a lower SDK version, if we remove the experiment here it will return `false`.
Do we want to return `true` instead?

Reply:
Good point, I tried to keep this in existing sync with the logic in other places. Might be better to create the function you mentioned above.